src/applications/facility-locator/components/search-results/LocationPhoneLink.jsx

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable prettier/prettier */
 /* eslint-disable no-use-before-define */
```

Comment:
This one is the linter not liking the order of declarations. `renderPhoneNumber` is a support function and below the SFC `LocationPhoneLink`, however, it is used a few times inside the main body of the SFC.
Fixing it would require the helper be moved up to the top of the file which would be confusing as this is the `LocationPhoneLink` component so that should be first.

It doesn't actually break the code or functionality of the component so I disabled it. (Near as I can tell this rule is only checking where in the file the function was declared and where it was used (i.e. line#). But no actual checks on whether or not it would actually cause a problem.)